content/docs/tech-insights/track-react-version/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Tracking Dependency versions
+publishedDate: '2024-01-08T15:15:00.0Z'
+description: How to track and visualize the dependency versions of your software with Roadie Tech Insights
+---
+
+Tracking the dependency version is critical for ensuring your software is secure and compliant. It can be difficult to track if your software is using a supported version, and which are using an End of Life version.
+
+With Roadie’s Tech Insights feature, there’s an easier way. In this tutorial, we’re going to:
+
+1. Automatically scan Components in the Backstage catalog and record which React version they’re using.
+2. Visualise the distribution of React versions being used.
+3. Create a check that can tell us which software is not using the supported version.
+
+As we go through this process, you’ll learn:
+
+1. How to create Data Sources, Checks and Scorecards with Tech Insights.
+2. How to parse files with Regex in Roadie and extract facts from those files.
+3. How to detect which repositories contain certain files like the package.json with Tech Insights.
+4. How to use the “Satisfies semantic version” operator to do comparisons.
+
+Let’s get started.
+
+## Record the React version automatically
+
+In order to track the React versions used in the catalog, we are going to create a Data Source In Roadie Tech Insights. It will be setup to run on a scheduled basis and extract the React version specified in the Package.json file.
```

Comment:
```suggestion
In order to track the React versions used in the catalog, we are going to create a Data Source In Roadie Tech Insights. It will be setup to run on a scheduled basis and extract the React version specified in the package.json file.
```